logger/logger.go

```diff
@@ -0,0 +1,74 @@
+package logger
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/databricks/databricks-sdk-go/logger"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
+type TfLogger struct {
+	Name string
+}
+
+// This function is always enabled because TfLogger implements the Logger interface from Go SDK and there we check
+// if the logging is enabled based on level (which default to Info). This however isn't required here
+// since we use the tflog package which automatically reads from TF_LOG environment variable.
```

Comment:
It isn't that this is not required. `tflog` simply does not expose what log level is supported for a logger. If it did, we would delegate to `tflog`.